neon_stt_plugin_scribosermo/__init__.py

```diff
@@ -49,6 +52,46 @@ def __init__(self, *args, **kwargs):
         self.model = tflit.Model(self.checkpoint_file)
         self.interpreter = self.model.interpreter
 
+        LOG.info(f"Scribosermo STT plugin loaded - model: {self.checkpoint_file}")
+
+    @property
+    def base_folder(self):
+        path = join(xdg.BaseDirectory.xdg_data_home, self.__class__.__name__)
+        makedirs(path, exist_ok=True)
+        return path
+
+    def download_scorer(self):
```

Comment:
consider converting `download` methods to functions outside the class for simpler unit tests